lib/config.py

```diff
@@ -119,19 +119,23 @@
         dict(help='Those environments will be available at the "Sofware '
              'selection" screen, along with the ones that come from the base '
              'distro (e.g. CentOS) repository. They will contain a group with '
-             'the same name, that will in turn contain the packages listed.'),
-    ('--base-distro-minimal-install-groups',):
-        dict(help='Packages groups that must be present in every base distro '
-             '(e.g. CentOS) installation and will be added to every '
-             'environment.', nargs='*'),
-    ('--iso-root-fs-packages-groups',):
-        dict(help='Packages groups that will be installed in the ISO root '
-             'file system. Groups implicitly created from the '
-             '"--installable-environments" option are automatically added to '
-             'this list.', nargs='*'),
-    ('--iso-root-fs-packages',):
-        dict(help='Packages that will be installed in the ISO root file '
-             'system, in addition to the specified groups.', nargs='*'),
+             'the same name, that will in turn contain the packages listed. '
+             'Those groups must be available in the ISO yum repository in '
+             'order to show up.'),
+    ('--iso-repo-minimal-packages-groups',):
+        dict(help='Packages groups that must be available in the ISO yum '
+             'repository and will always be installed in the target system.',
+             nargs='*'),
```

Comment:
In practice, what is the difference between adding packages/groups using these options vs. using the 'additional' ones?